server/presenters/integration.js

```diff
@@ -0,0 +1,8 @@
+// @flow
+import { Integration } from '../models';
+
+function present(ctx: Object, integration: Integration) {
+  return integration.toJSON();
```

Comment:
In general I would prefer at least pulling the keys out so that we don't accidentally expose data in the API. I been bitten by these before :(